src/components/website-management/website-form-components/customization/customization-components/CookieBanner.js

```diff
@@ -0,0 +1,271 @@
+
+    /*Sanck bar useStates*/
+    const [snackbarOpen, setSnackbarOpen] = useState(false);
+
+    const [snackbarMessage, setSnackbarMessage] = useState('');
+
+    const [error, setError] = useState(false);
+
+    const [errorMessage, setErrorMessage] = useState('');
+
+    const [showDialog, setShowDialog] = useState(false);
+
+    const handleCloseClick = () => {
+        setShowDialog(false);
+    };
+
+    const handleCustomizeClick = () => {
+        setShowDialog(prevShowDialog => !prevShowDialog);
+    };
+
+    console.log("passed id"+passedId);
+
+
+
+    const loadTemplates = async () => {
+
+        const websiteID = passedId;
+        const result = await axios.get(`http://localhost:8080/api/template/getAllTemplates/${websiteID}`);
+        settemplates(result.data);
+
+        if(Object.keys(selectedTemplate).length !== 0){
+            const template = result.data.find((t) => t.templateRegulation === selectedTemplate?.templateRegulation);
+            setSelectedTemplate(template);
+        } else {
+            const template = result.data[0];
+            console.log("Empty Value : ", template)
+            setSelectedTemplate(template);
+        }
+    }
+
+
+    const laodBanner = async () => {
+        const websiteID = passedId;
+        try {
+            const response = await axios.get(`http://localhost:8080/api/banner/getBanners/${websiteID}`);
+            const result = response.data;
+            setBanner(result);
+
+            const { bannerPosition, bannerColor, bannerAlignment } = result;
+            setPositionValue(bannerPosition);
+            handleChangeTop(bannerPosition);
+            handleChangeHorizontal(bannerAlignment);
+            handleChangeColor(bannerColor);
+        }
+        catch (error) {
+            console.error("Error loading banners:", error);
+        }
+    };
+
+
+    console.log("this is banner"+JSON.stringify(banner));
+    console.log("this is alignment"+alignmentValue);
+
+    console.log("this is color"+color);
+    console.log("this default"+positionValue);
+
+
+
+    useEffect(() => {
+        loadTemplates();
+        laodBanner();
+    }, []);
+
+
+    const regulationValues = templates?.length > 0 ? templates.map(template => ({ id: template.templateId, value: template.templateRegulation }))
+        : [];
+
+    const choosenTemplate = (e) => {
+        const selectedRegulation = e.target.value;
+        let chosenTemp=null;
+        let chosenCont=null;
+
+        for (let i = 0; i < templates.length; i++) {
+            if (templates[i].templateRegulation === selectedRegulation) {
+                chosenCont= templates[i].templateContent;
+                break;
+            }
+        }
+
+        const template = templates.find((t) => t.templateRegulation === e.target.value);
+        setSelectedTemplate(template);
+        setChosenCont(chosenCont);
+    };
+
+    function handleChangeTop(value){
+        if(value==="Top" ){
```

Comment:
Format the other places as well. 

```suggestion
        if(value === "Top" ){
```

Reply:
fixed